src/magic_folder/migrate.py

```diff
@@ -81,7 +81,8 @@ def magic_folder_migrate(config_dir, listen_endpoint_str, tahoe_node_directory,
             author,
             mf_config[u'collective_dircap'],
             mf_config[u'upload_dircap'],
-            mf_config[u'poll_interval'],  # is this always available?
+            int(mf_config[u'poll_interval']),  # is this always available?
+            int(mf_config[u'poll_interval']),
```

Comment:
should this maybe be `scan_interval` ?

Reply:
tahoe-lafs didn't have that (which is where this is migrating from. I've added a comment to clarify.